nosetests_answers.ini

```diff
@@ -15,4 +15,5 @@ author.irc = irc.freenode.org#plone
 
 plone.version = 4.3.6
 plone.is_plone5 = False
+plone.minor_version = 4.3
```

Comment:
I think we can extract the minor version from plone.version without adding an additional question.